Add "Reform" as a search topic

NathanLewis opened the pull request:
The suggestion to do this came up in the Understanding Reform hackathon

NathanLewis committed:
Added "Reform" as a topic

tyronen commented on backend/chalicelib/topics.py:
```diff
@@ -76,11 +76,16 @@
         "delays train",
     ],
+    "Reform": [
+        "abortion",
```
Won't this put up quite a bit of noise? stories unrelated to the party

NathanLewis replied:
I tested it locally. It wasn't bringing in a lot of noise but it was pulling in some US stories so I removed "abortion"

NathanLewis commented:
That suggestion came from a Labour campaigner who is very active in her CLP
in the North East (England). I was a bit surprised at this suggestion too
but she said this is an issue Reform are taking on. I'll test it locally
here and see if the results seem noisy or not.


On Wed, 4 Jun 2025 at 00:09, Tyrone Nicholas ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In backend/chalicelib/topics.py
> <https://github.com/CampaignLab/localnews-finder/pull/11#discussion_r2125089175>
> :
>
> > @@ -76,11 +76,16 @@
>          "cancellations UK trains",
>          "delays train",
>      ],
> +    "Reform": [
> +        "abortion",
>
> Won't this put up quite a bit of noise? stories unrelated to the party
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CampaignLab/localnews-finder/pull/11#pullrequestreview-2894345503>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAODYFUGIR3Y4ZEE2IW3JF33BYTJLAVCNFSM6AAAAAB6LEP4WSVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDQOJUGM2DKNJQGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

NathanLewis committed:
Update topics.py

Removed "abortion" from "Reform". It wasn't causing a lot of noise but it did have the effect of pulling in US stories.
